front/temporal/scrub_workspace/activities.ts

```diff
@@ -153,6 +153,7 @@ async function deleteDatasources(auth: Authenticator) {
   // First, we delete all the data source views.
   const dataSourceViews = await DataSourceViewResource.listByWorkspace(auth);
   for (const dataSourceView of dataSourceViews) {
+    // Soft delete the data source view.
```

Comment:
What is in charge of hard deleting it?

Reply:
The call to `deleteDataSource` right below on line 166.